streamparse/ext/invoke.py

```diff
@@ -99,26 +99,30 @@ def list_topologies(env_name="prod"):
         return _list_topologies()
 
 
-def _kill_topology(topology_name, run_args=None, run_kwargs=None):
+def _kill_topology(topology_name, wait, run_args=None, run_kwargs=None):
     if run_args is None:
         run_args = []
     if run_kwargs is None:
         run_kwargs = {}
     run_kwargs['pty'] = True
-    cmd = ["lein",
-           "run -m streamparse.commands.kill_topology/-main",
-           topology_name]
-    return run(" ".join(cmd), *run_args, **run_kwargs)
+    wait_arg = ("--wait %s" % wait) if wait is not None else ""
+    cmd = ("lein run -m streamparse.commands.kill_topology/-main"
+           " %(topology_name)s %(wait)s") % \
```

Comment:
Please use `.format` not `%` for string formatting, just convention we've been using throughout streamparse.